specification/sql/resource-manager/Microsoft.Sql/preview/2021-08-01-preview/examples/ManagedDatabaseSensitivityLabelsListByDatabase.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "myRG",
+    "managedInstanceName": "myManagedInstanceName",
+    "databaseName": "myDatabase",
+    "api-version": "2021-08-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/myRG/providers/Microsoft.Sql/managedInstances/myManagedInstanceName/databases/myDatabase/schemas/dbo/tables/myTable/columns/myColumn/sensitivityLabels/current",
```

Comment:
>Sql [](http://example.com/codeflow?start=111&length=3)

your examples do not seem to match the api changes that you have made. Can you please update to fix this ?

Reply:
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/sensitivityLabels"

Reply:
@raosuhas 
this is expected. 
The Id is based on current sensitivity labels on the column. not the database. the column is the fine granularity with sensitivity labels. 

If you have additional questions, please contact me at ji_dot_wang_at_MSFT

Reply:
ping @raosuhas again~ thanks~

Reply:
OK that makes sense. Looks like this is a LIST api modelled at a higher level for the sensitivityLabel. 

Sorry I misread this in the first instance. Generally we expect that the collection is already implemented in the first iteration and is supported at the immediate higher level. 

i.e this needs to have a support to get the collection of the sensitivityLabel at the immediate parent level : i.e you should support something like this : 

GET /subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/myRG/providers/Microsoft.Sql/managedInstances/myManagedInstanceName/databases/myDatabase/schemas/dbo/tables/myTable/columns/myColumn/sensitivityLabels


This is documented at : [https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy008-support-a-collection-get-and-individual-get-for-child-resource-types-even-singletons](https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy008-support-a-collection-get-and-individual-get-for-child-resource-types-even-singletons) 

Please add that API as well. If you would prefer to discuss over an email you can send an email to armapireview

Thanks !

Reply:
@raosuhas what is your msft alias? can I directly ping you? this communication loop is too long. thanks! or you can directly ping me. thanks~